packages/js/settings-editor/src/route.tsx

```diff
@@ -0,0 +1,190 @@
+/**
+ * External dependencies
+ */
+import {
+	createElement,
+	useEffect,
+	useMemo,
+	useState,
+} from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { getQuery } from '@woocommerce/navigation';
```

Comment:
I think we should use `@wordpress/router` instead of WooCommerce navigation methods, which makes assumptions about the route. This is its own standalone page outside of wc-admin. I think it would be best to keep the routes the same as they are now, ie `wp-admin/admin.php?page=wc-settings&tab=shipping`

Reply:
I add `"@wordpress/router": "1.11.0",` in https://github.com/woocommerce/woocommerce/pull/52691